example.config.json

```diff
@@ -17,5 +17,20 @@
 			"role id"
 		]
 	},
+	"stripe": {
+		"secret_key": "sk_secret",
+		"webhook_secret": "whsec_secret"
+	},
+	"database": {
+		"account": {
+			"address": "127.0.0.1",
+			"port": 27017,
+			"database_name": "pretendo",
```

Comment:
You can technically use the address to force username and password in the generated url. But it's better to allow for inputting them, or simply just have a `connection_string` in the config. (Which is the recommended way by mongo iirc)

Reply:
Yeah I have it use a full URI in the account server. Not sure why I did it this way here, my bad

Reply:
fixed